RELEASE_NOTES

```diff
@@ -11,6 +11,7 @@
     - ParsedSql can be created with ParsedSql.of(String, ParsedParameters) factory
     - ParsedParameters can be created with ParsedSql.positional(int) and
       ParsedSql.named(List<String>) factories.
+    - SqlArrayType.of(String, Function) / SqlArrayTypeFactory.of(Class, String, Function) factory methods
```

Comment:
is this enough information? Seems like a note only frequent users of SqlArrayType will find meaning in